packages/indexer-cli/src/commands/indexer/disputes/get.ts

```diff
@@ -6,8 +6,7 @@ import { createIndexerManagementClient } from '../../../client'
 import {disputes, printDisputes} from '../../../disputes'
 
 const HELP = `
-${chalk.bold('graph indexer disputes get')} [options] all
-${chalk.bold('graph indexer disputes get')} [options] <status>
+${chalk.bold('graph indexer disputes get')} [options] <status> <minimumAllocationClosedEpoch>
```

Comment:
What are the supported status values? I think it's worth documenting them. And in case they are `"Potential"` etc. we should make them all lowercase because that's more ergonomic.

Reply:
`potential`, `valid`, and `pending` but only `potential` is used at the moment.  

I've updated to use lowercase `potential` which I agree is preferable.

Reply:
Should we document the potential status values in the help text?